src/main/java/org/jmhsrobotics/frc2024/Constants.java

```diff
@@ -85,7 +85,7 @@ public static class SwerveConstants {
 		/* START DRIVER CONFIGURABLES */
 
 		// configure to change drivetrain turbo mode
-		public static final double kMaxSpeedFeetPerSecond = 14;
+		public static final double kMaxSpeedFeetPerSecond = 16;
 
 		// configure to change drivetrain slow mode
 		public static final double slowSpeedFeetPerSecond = 12; // TODO: Remove used when not in turbo mode move to Comp
```

Comment:
Was slow mode also requested to be increased?

Reply:
i'll confirm it with Annabelle today